### DISCUSSION TITLE
Bulk submit command for automation

### DISCUSSION THREAD

**Discussion by KarbitsCode** [microsoft/winget-create#568] [Q&A] [answered]
I just want to ask if the `wingetcreate submit` command can set a custom md or text to PR bodies, like automatically setting the PR checklist so I don't have to manually edit it each time it's open.

Also, is there an option to not open the PR page after the submit command? That will be very helpful.

**Comment by mdanish-kh** [comment, marked as answer]
winget-create doesn't have these capabilities yet. Feel free to open feature requests on the issue tracker so these requests can be tracked more easily

**Comment by mdanish-kh** [comment]
Just for tracking purposes: This was shipped in WinGet-Create 1.12 release with `--no-open` CLI arg and a user setting to control default behavior
